clap-utils/src/keypair.rs

```diff
@@ -69,7 +69,12 @@ pub fn keypair_from_seed_phrase(
         let passphrase = prompt_passphrase(&passphrase_prompt)?;
         keypair_from_seed_phrase_and_passphrase(&seed_phrase, &passphrase)
     } else {
-        let mnemonic = Mnemonic::from_phrase(seed_phrase, Language::English)?;
+        let sanitized = seed_phrase
+            .split(' ')
+            .filter(|word| !word.is_empty())
+            .collect::<Vec<&str>>()
+            .join(" ");
```

Comment:
Maybe `split_ascii_whitespace()` and `fold()`?
https://doc.rust-lang.org/std/primitive.str.html#method.split_ascii_whitespace

Reply:
Sounds good, I think https://doc.rust-lang.org/std/primitive.str.html#method.split_whitespace would be better though, what do you think?

Reply:
Lol, sounds good, I almost put that first

Reply:
Opted to keep `join` over `fold` because `join` makes it easier to add a delimiter